src/Shape/Shape.js

```diff
@@ -44,6 +44,7 @@ export default class Shape extends BaseClass {
         return s * 3;
       }
     };
+    this._ariaLabel = "";
```

Comment:
make the default value use the constant function: `this._ariaLabel = constant("");`